tests/string/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -120,11 +124,14 @@ $string = "This is the value of the var named by the return value of getName():
     $arg3
 )}";
 // $string = "This is the value of the var named by the return value of \\$object->getName(): {\${$object->getName()}}";
-$string = "I'd like an \${beers::softdrink}
-";
-$string = "I'd like an \${beers::$ale}
-";
+$string = "I'd like an \${beers::softdrink}\\n";
+$string = "I'd like an \${beers::$ale}\\n";
 $test = "\\\\";
 $test = '\\\\';
+printf("Query run in $queryTimeTaken seconds.\\n");
+$test =
+    "You can also have embedded\\n newlines in
+strings this way as it is
+okay to do";
```

Comment:
Indentation is strange here, right? (Has not much to do with this PR though)

Reply:
agreed. i think thats just the way our `assign` node breaks though if we go over the line limit. maybe we need to think about an exception for strings or something? not sure best way to handle it

Reply:
Yep, that might be an idea. Let's open a new issue about this, it's probably not very urgent.